ufirebase.py

```diff
@@ -3,10 +3,16 @@
 import ussl
 import _thread
 import time
+import urllib.urequest
```

Comment:
SSLContext not sup. in urequest
see ssl.SSLContext()](https://docs.micropython.org/en/latest/library/ssl.html)

Reply:
urllib.urequest is from pycopy https://github.com/pfalcon/pycopy-lib

I am using a slightly modified version of the urequest due to the SSLContext()

Will commit it too, i completely overlooked that

Reply:
Just a thing to note is that I am not using the full pycopy lib as space on my ESP32 is limited, i am only specifically using urllib.urequest and modified it for my own purposes.

For my purposes the workaround for the SSLContext issue works fine, might want to opt for a permanent solution or a fork of pycopy with the same modifications if you are interested in using the entire library